src/modules/bot/bot.controller.ts

```diff
@@ -125,6 +125,7 @@ export class BotController {
     @Query('startingMessage') startingMessage: string,
     @Query('match') match: 'true' | 'false',
     @Query('sortBy') sortBy: string | undefined,
+    @Query('orderBy') orderBy: string | undefined,
```

Comment:
Hey instead of defining `orderBy` as `string`, we should define it as `'asc' | 'desc' | 'undefined'` since these are the only two values.